<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DllDemo2Class</class>
 <widget class="QMainWindow" name="DllDemo2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DllDemo2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>实时视频</string>
    </property>
    <widget class="QListWidget" name="relListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="relcmidEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>201201</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cameraid</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regRelButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unRegRelButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
    <widget class="QLabel" name="relInfoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>351</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>历史视频</string>
    </property>
    <widget class="QListWidget" name="hisListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cameraid</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hiscmidEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>201201</string>
     </property>
    </widget>
    <widget class="QPushButton" name="regHisButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>开始时间戳</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="begtimeEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1482891846</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>结束时间戳</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="endtimeEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1482892400</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unRegHisButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
    <widget class="QLabel" name="hisInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>351</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>0</y>
      <width>801</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>初始化</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ip:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.3.192</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8080</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>user:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>pwd:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Haibo2013</string>
     </property>
    </widget>
    <widget class="QPushButton" name="initButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
